backend/src/config.ts

```diff
@@ -101,4 +109,11 @@ export const sourceChains = [
   },
 ];
 
+export const archives = [
+  {
+    url: "wss://kusama-rpc.polkadot.io",
+    path: "/Users/apple/Documents/kusama-archive-2021-oct-23",
+  }
+]
```

Comment:
Why is this not an environment variable or something like that?

Reply:
this was initially env variable, but what if we have 10+ archives?

Reply:
Then config file? Should be better than hardcoded value at least.

Reply:
I thought of this as a config file :) what would you recommend?

Reply:
Well, the config file should be external. I think at this point we have a few config files that can be configured, so let's add them to readme and make this list of archives also a config file that is read from somewhere.